### PR TITLE
Upgrade the mdn-confluence dependency for new/better defaults

### PR DESCRIPTION
**Pull request by foolip**
This brings in https://github.com/mdittmer/mdn-confluence/pull/31.

### CODE REVIEW

**Comment by foolip** [comment]
@jpmedley can you try this out and review? As you'll see, the version is no longer pinned, although the exact commit used will be recorded in package-lock.json. This means that future changes to the master branch of mdn-confluence will propagate where before an npm release was required. I feel this is an improvement, and don't expect the mdn-confluence repo will change all that much.

**Comment by vinyldarkscratch** [comment]
Personally, I think that keeping the updates to `npm` rather than linking directly to the Git repository is the better solution.  In most repositories, a Git's master branch is not meant for production, and may include bugs or crashes.  Plus, `npm` is designed specifically to manage Node.js packages like this, and a modification to switch away from using it feels awkward.

I suggest simply adding a `^` before the version number, so that `mdn-confluence` will always be at the latest version released via `npm`.

File: package.json [hunk 1]
```diff
@@ -29,7 +29,7 @@
     "ajv": "^6.6.2",
     "better-ajv-errors": "^0.6.1",
     "compare-versions": "^3.4.0",
-    "mdn-confluence": "0.0.7",
+    "mdn-confluence": "git://github.com/mdittmer/mdn-confluence.git",
```
Comment (vinyldarkscratch) [inline package.json]:
```suggestion
    "mdn-confluence": "^0.0.7",
```

**Comment by foolip** [comment]
@vinyldarkscratch note that it would be me taking care of the releases to npm, and I'd probably set it up to publish automatically.

@mdittmer are you able to add me as a collaborator on the npm package so I can cut a new release?

**Comment by jpmedley** [comment]
@foolip I'll try to get to this today.

**Comment by foolip** [comment]
I've published https://www.npmjs.com/package/mdn-confluence/v/1.0.0 and made that the dependency now.

**Comment by foolip** [comment]
Yes, I'm quite aware of that, and have been trying to remove some. It depends on confluence in turn which is responsible for most of it, so I'll see if I can break that dependency.

**Pull request merged**